styles/style.css

```diff
@@ -169,3 +174,96 @@ section {
         justify-content: center;
     }
 }
+
+footer {
+    position: sticky;
+    bottom: 0;
+    padding: 20px;
+    background-color: #082264;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+    font-size: 16px;
+    line-height: 20px;
+}
+
+footer p {
+    color: white;
+    margin: 0;
+}
+
+footer p a {
+    color: white;
+}
+
+.content .container {
+    width: auto;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+}
+
+.content:nth-child(even) .container {
+    flex-direction: row-reverse;
+}
+
+.content .container img {
+    width: 100%;
+    height: auto;
+    max-width: 518px;
+    max-height: 350px;
+}
+
+.contentText {
+    max-width: 523px;
+}
+
+.contentText h3 {
+    font-style: normal;
+    font-weight: 600;
+    font-size: 18px;
+    line-height: 22px;
+    color: #55B8F0;
+}
+
+.contentText h2 {
+    margin: 0;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 36px;
+    line-height: 44px;
+    color: #082264;
+}
+
+.contentText p {
+    font-style: normal;
+    font-weight: normal;
+    font-size: 18px;
+    line-height: 22px;
+    color: #082264;
+}
+
+.contentText button {
```

Comment:
Try applying styles to class (e.g. ".btn-primary") instead of tag inside some wrapper.